test/snowpack-modules/package.json

```diff
@@ -2,7 +2,7 @@
   "type": "module",
 
   "scripts": {
-    "test": "../node_modules/.bin/snowpack build; node build/index.js"
+    "test": "..\\node_modules\\.bin\\snowpack build && node build/index.js"
```

Comment:
... turns out none of these tests actually ran on a windows machine - they all silently did nothing. (except webpack 5, which didn't use a forward slash). Backslashes and `&&` instead of `;` are portable enough to work on both a windows and linux shell, given the tools we're using.

Related: webpack 5 tests fail on any node version newer than 16 - they apparently rely on an old open ssl feature in the crypto they use. If we upgrade our `node` test matrix, we'll have to update the webpack 5 test as well 9since I'm sure it's fixed in later versions of webpack - `5.0.0-rc.4` is... not new).

Reply:
I didn't think you could use `\` as a path separator in Linux as it treats it as a literal character. I've always found it safer to use `/`.

Reply:
I guess this is an outdated commit anyways since the path qualification wasn't even necessary.

Reply:
Yeah, I ended up removing the qualification entirely and updating npm instead.